### ISSUE TITLE
README is out of date as its instructions does not work for Linux anymore.

### ISSUE THREAD

**Issue by kgardas** [pharo-spec/Spec-Gtk#5]
Attempting to follow install instructions for Linux I've ended with `Could not resolve: Spec2-CommonWidgets [Spec-2CommonWidgets] in .../pharo-local/package-cahce github://pharo-spec/Spec:v1.0.2/src` error message. This is while running the Metacello code load snipped.

**Comment by riverdusty** [comment]
Occurs on Windows 10 while loading into Pharo9.0.

![image](https://user-images.githubusercontent.com/572263/158170594-a1ddb207-5d3a-4e18-99d3-75f7a0416017.png)

**Comment by hoemac** [comment]
Yes riverdusty, I confirm, Spec-Gtk is not loadable in current Pharo 9.
But do you think this is the right place to report the problem ?
Hide it here in an old issue from Sep 21 that no one is assigned to ?
Shouldn't we create a new issue ?

**Comment by hoemac** [comment]
You need Pharo 10 to load Spec-Gtk, is too bleeding edge to has sense in older Pharo versions.

Esteban
